astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -401,7 +401,7 @@ def test_icrs_altaz_moonish(testframe):
     right AltAz distance
     """
     # we use epv00 instead of get_sun because get_sun includes aberration
-    earth_pv_helio, earth_pv_bary = epv00(testframe.obstime.jd1, testframe.obstime.jd2)
+    earth_pv_helio, earth_pv_bary = epv00(testframe.obstime.tdb.jd1, testframe.obstime.tdb.jd2)
```

Comment:
Here again the `obstime_tdb = obstime if scale == 'tdb' else obstime.tdb` stanza.